### PR TITLE
updated matrix_power function

### PR DESCRIPTION
**Pull request by ndurumo254**

### CODE REVIEW

**Comment by ndurumo254** [comment]
> _ No description provided. _
formatted the task

**Comment by ndurumo254** [comment]
in case where docker fails, is it advisable to use the venv only?

**Comment by ndurumo254** [comment]
hello any response to the above pull request?

**Comment by kimbring2** [comment]
@ndurumo254 Hello, you need to write the test function also. Check the [link](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_linalg.py) for example.

**Comment by ndurumo254** [comment]
> @ndurumo254 Hello, you need to write the test function also. Check the [link](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_linalg.py) for example.

thanks kimbring2 for your response. here is the updated version

**Comment by ndurumo254** [comment]
Am confused on how i should activate this pull request.  i cant see the place to request for review.

**Comment by kimbring2** [comment]
@ndurumo254 Thank you for adding the test code. But, you should add code to only one of the Jax frontends. What was your issue when opening this PR? Was it the jax.lax or jax.numpy?

**Comment by ndurumo254** [comment]
> @ndurumo254 Thank you for adding the test code. However, you should add code to only one part of the Jax frontend. What was your issue when opening this PR? Was it the jax.lax or jax.numpy?

@kimbring2. thanks for reviewing. my issue was on lax.numpy.  I have counter-checked the request and saw I had mixed-up issues. I have however rectified that. looking forward to getting more direction from you.

**Comment by kimbring2** [comment]
@ndurumo254 Yes, it seems like the code location is corrected well. However, you need to change the argument name according to [the actual function document](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.matrix_power.html).

The purpose of test is that output of Ivy function is same to original own.

**Comment by ndurumo254** [comment]
> @ndurumo254 Yes, it seems like the code location is corrected well. However, you need to change the argument name according to [document of the actual function](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.matrix_power.html).
> 
> The purpose of the test function is that the output of the Ivy function is the same as the output of the original function.

@kimbring2. 
changed

**Comment by ndurumo254** [comment]
> > @ndurumo254 Yes, it seems like the code location is corrected well. However, you need to change the argument name according to [document of the actual function](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.matrix_power.html).
> > The purpose of the test function is that the output of the Ivy function is the same as the output of the original function.
> 
> @kimbring2. changed
@kimbring2 Hello kimbring2, any idea on how i can get out of this error "test-frontend-jax / run-nightly-tests"?

**Comment by kimbring2** [comment]
@ndurumo254 Hello, actually you do not need to care about that error. You should run the pytest for the test_jax_numpy_matrix_power function.

**Comment by ndurumo254** [comment]
> @ndurumo254 Hello,
> 
> Actually, you do not need to care about that error. You should run the pytest for the test_jax_numpy_matrix_power function.
> 
> How do you run the test now? See the [document](https://lets-unify.ai/ivy/deep_dive/15_ivy_tests.html) for that.



> @ndurumo254 Hello,
> 
> Actually, you do not need to care about that error. You should run the pytest for the test_jax_numpy_matrix_power function.
> 
> How do you run the test now? See the [document](https://lets-unify.ai/ivy/deep_dive/15_ivy_tests.html) for that.

@kimbring2 hello. any time i try to run the test i get this error. No module named 'pytest'. this is even after installing pytest module manually.  would you kindly help me

**Comment by ndurumo254** [comment]
> > @ndurumo254 Hello,
> > Actually, you do not need to care about that error. You should run the pytest for the test_jax_numpy_matrix_power function.
> > How do you run the test now? See the [document](https://lets-unify.ai/ivy/deep_dive/15_ivy_tests.html) for that.
> 
> > @ndurumo254 Hello,
> > Actually, you do not need to care about that error. You should run the pytest for the test_jax_numpy_matrix_power function.
> > How do you run the test now? See the [document](https://lets-unify.ai/ivy/deep_dive/15_ivy_tests.html) for that.
> 
> @kimbring2 hello. any time i try to run the test i get this error. No module named 'pytest'. this is even after installing pytest module manually. would you kindly help me

@kimbring2 . i cant see the progress  chart of failing and passing tests. what would be the problem

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Auxia** [comment]
Hi @ndurumo254, are you still working on this PR? It seems to have gone stale, please let me know if you are still working on it and need help.
